src/everest/config/optimization_config.py

```diff
@@ -8,7 +8,7 @@
 
 class OptimizationConfig(BaseModel, extra="forbid"):  # type: ignore
     algorithm: str | None = Field(
-        default="default",
+        default="dakota/default",
```

Comment:
If that works, that's a coincidence :-) Maybe we should say `default=optpp_q_newton" so that we really fix the default of Everest to that.